RecoLocalCalo/HGCalRecProducers/plugins/alpaka/HGCalLayerClustersAlgoWrapper.dev.cc

```diff
@@ -0,0 +1,38 @@
+// Check that ALPAKA_HOST_ONLY is not defined during device compilation:
+#ifdef ALPAKA_HOST_ONLY
+#error ALPAKA_HOST_ONLY defined in device compilation
+#endif
+
+#include "RecoLocalCalo/HGCalRecProducers/interface/HGCalTilesConstants.h"
+
+#include "HGCalLayerClustersAlgoWrapper.h"
+
+#include "CLUEAlgoAlpaka.h"
+
+namespace ALPAKA_ACCELERATOR_NAMESPACE {
+
+  using namespace cms::alpakatools;
+
+  void HGCalLayerClustersAlgoWrapper::run(Queue& queue,
+                                          const unsigned int size,
+                                          const HGCalSoACellsDeviceCollection::ConstView inputs,
+                                          HGCalSoACellsOutDeviceCollection::View outputs) const {
+    CLUEAlgoAlpaka<ALPAKA_ACCELERATOR_NAMESPACE::Acc1D, Queue, HGCalSiliconTilesConstants, 96> algoStandalone(
```

Comment:
what is `96` ?

Reply:
It's the total number of HGCAL layers, cumulative between positive and negative z-side.

Reply:
Is it defined anywhere as a constant ?